setup.py

```diff
@@ -63,7 +63,7 @@ def check_setuptools_features():
     # TODO Consider not installing the db drivers, or putting them in extras.
     'rethinkdb~=2.3',  # i.e. a version between 2.3 and 3.0
     'pymongo~=3.4',
-    'pysha3==1.0.0',
+    'pysha3==1.0.2',
```

Comment:
Yeah... not sure if there was a reason behind pinning it in the first place, but `~=` sounds good to me. @libscott do you think we should keep pinning it?